examples/Identifying Local Environments in a Complex Crystal.ipynb

```diff
@@ -24,7 +24,9 @@
    "cell_type": "code",
```

Comment:
<p>Let's replace this block with the following:</p><p> </p><p>For the <code>cluster_neighbors</code> parameter, which specifies the number of neighbors the environment cluster of each particle will be compared with, we choose a neighbor list containing all other particles in the system. Since this system contains only 416 particles, comparing the environments of every single pair of particles does not slow down the computation significantly. To quickly and easily build such a neighborlist, we use the <code>all_pairs</code> factory method of the <code>NeighborList</code> class.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>